lib/Sema/TypeCheckDecl.cpp

```diff
@@ -1785,51 +1785,41 @@ static void checkGenericParamAccess(TypeChecker &TC,
     return;
 
   // This must stay in sync with diag::generic_param_access.
-  enum {
-    ACEK_Parameter = 0,
-    ACEK_Requirement
+  enum ACEK {
+    Parameter = 0,
+    Requirement
```

Comment:
This pollutes the surrounding namespace unless the enum is declared with `enum class`.

Reply:
Ah, true. It is OK to commit the change here?

Reply:
You can open separate PR for to correct this, it's not a problem!